problem_builder/dashboard.py

```diff
@@ -336,20 +372,23 @@ def student_view(self, context=None):  # pylint: disable=unused-argument
                 'display_name': mentoring_block.display_name,
                 'mcqs': []
             }
-            for child_id in mentoring_block.children:
-                if child_isinstance(mentoring_block, child_id, MCQBlock):
-                    # Get the student's submitted answer to this MCQ from the submissions API:
-                    mcq_block = self.runtime.get_block(child_id)
-                    mcq_submission_key = self._get_submission_key(child_id)
-                    try:
-                        value = sub_api.get_submissions(mcq_submission_key, limit=1)[0]["answer"]
-                    except IndexError:
-                        value = None
-                    block['mcqs'].append({
-                        "display_name": mcq_block.display_name_with_default,
-                        "value": value,
-                        "color": self.color_for_value(value) if value is not None else None,
-                    })
+            hide_questions = self.exclude_questions.get(mentoring_block.url_name, [])
```

Comment:
You ought to add in some validation here, in case someone puts in data in an unreadable format (for example swaps [] for the { } or something).